package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "edx-ux-pattern-library",
-  "version": "0.2.0",
+  "version": "0.1.46",
```

Comment:
Double also, this file should be managed by your local node. Did you do a ``npm install --save-dev`` or a ``npm uninstall --save-dev``?